pyinfra/facts/files.py

```diff
@@ -354,3 +356,21 @@ class FindDirectories(FindFilesBase):
     """
 
     type_flag = "d"
+
+
+class Flags(FactBase):
+    """
+    Returns a list of the file flags set for the specified file or directory.
+    """
+
+    requires_command = "chflags"  # don't try to retrieve them if we can't set them
+
+    def command(self, path):
+        return make_formatted_string_command(
+            "stat -f %Sf {0} || true",
```

Comment:
```suggestion
            "! test -e {0} || stat -f %Sf {0}",
```

Fact have slowly been removing the use of `|| true` because it hides any error - this way we either do nothing if no file or run the `stat` command if the file exists.

Reply:
that makes sense. change made.